dev/io.openliberty.jakarta.security.2.0/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.security.2.0; singleton:=true
+
+Export-Package: \
+    jakarta.security.enterprise;version="1.0.0", \
```

Comment:
Should these be 2.0.0 instead of 1.0.0?

Reply:
They are new packages. They don't need to match the bundle version as far as I know.

Reply:
Changed these to be 2.0.0 instead of 1.0.0 and updated the jakarta-versions.properties file as well (to address your comment there).